_positions/visual-designer.md

```diff
@@ -1,40 +1,29 @@
-delight. Visual Designers will be well-versed in all aspects of current
-visual design standards and trends and will be responsible for managing
-project design reviews, resource planning, and execution for all project
-work related to visual design.
-
-Primarily responsible for:
-
--   Oversees all visual design efforts
-
--   Guides, mentors, and coaches team members while leading projects to
-successful completion
-
--   Develops and maintains relationships with key peers in Marketing,
-Branding, UX leaders, IT leaders, and others to identify and plan
-creative solutions
-
--   Manages external service resources and budgets for visual design
-
--   Ensures successful completion of all work executed by the team (on
-time, on budget, and ensuring quality)
-
--   Ensures compliance with the project management methodologies and the
-Project Management Office processes and standards
-
--   Develops, maintains, and ensures compliance of application release
-management, outage management and change control processes and
-standards
-
--   Defines, creates, communicates, and manages resource plans and other
-required project documentation such as style guides and provides
-updates as necessary
+##Role: Visual Designer
+
+###Position Summary
+Visual designers at 18F craft highly usable and beautiful web-based experiences for the benefit of the public. Because our clients are agencies of the U.S. government, our work should reflect and help preserve the integrity of our national image. Our visual designers must collaborate to establish and evolve design patterns which make our work consistent and efficient and can inform the design of websites for agencies who aren’t our clients. Additionally they should constantly push the boundaries and leverage the best and newest creative approaches and web technologies.
+
+###Key Objectives
+**Objective #1: You will deliver a high-quality, user-centered web-design in an agile environment.**
+- Successfully interpret critical notes from clients, design leaders, and team members.
+- Create elegant, consistent, and usable designs which artfully translate wireframes into pixels, and offer front end developers friction-free implementation.
+- Employ key design principles and master the requisite tools.
+- Respectfully champion the cause of the user despite occasional stakeholder opposition.
+- Foster cross-pollination with non-18F government designers.
+- Create designs within the parameters of Section 508 to ensure accessibility for all projects.
+- Tackle the job in front of you, regardless of whether it offers a significant opportunity for creative expression.
+- Grow a consistent and beautiful body of work, which garners unsolicited praise from clients and users alike.
+
+**Objective #2: You’ll maintain the 18F brand by implementing and expanding upon design patterns.**
+- Utilize 18F’s set of design patterns including palettes, form components, and other re-usable elements.
```

Comment:
From the rather opinionated [Smart Quotes for Smart People](http://smartquotesforsmartpeople.com/):

> When dealing with code, straight quotes are often required.